docs/ui/README.md

```diff
@@ -148,6 +148,17 @@ Flipper::UI.configure do |config|
 end
 ```
 
+We can also associate a `description` for each `feature` by providing a descriptions source:
+
+```ruby
+Flipper::UI.configure do |config|
+  config.descriptions_source = ->(keys) do
```

Comment:
This documentation doesn't match the code. It shows setting it to a block here, but is defaulted to a hash in ui config. If it were suppose to be a block, it should be defaulted to an empty block in ui config, right?